client/src/visualiser-src/linked-list-visualiser/animation-producer/LinkedListAnimationProducer.ts

```diff
@@ -2,15 +2,11 @@ import { SVG, Runner, Element } from '@svgdotjs/svg.js';
 import {
```

Comment:
We should probably make the file naming more consistent/come up with a rule for naming files.

Reply:
The React file naming rules are outlined in our style guide on Confluence.

I think hyphenated lowercase names for .ts and .js are preferred.